custom_components/google_home/__init__.py

```diff
@@ -76,7 +76,7 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
 
     # Offload the loading of entities to the platform
     for platform in PLATFORMS:
-        hass.async_add_job(
+        hass.async_create_task(
```

Comment:
Why is this change needed? 
from [docs](http://dev-docs.home-assistant.io/en/master/api/core.html):
> 
> async_add_job(target: Callable[[...], Any], *args: Any) → Optional[_asyncio.Future][source]
> Add a job from within the event loop.
> 
> This method must be run in the event loop.
> 
> target: target to call. args: parameters for method to call.
> 
> async_create_task(target: Coroutine) → _asyncio.Task[source]
> Create a task from within the eventloop.
> 
> This method must be run in the event loop.
> 
> target: target to call.

Reply:
It doesn't return `Callable`, I checked other places and they use `async_create_task`: https://github.com/home-assistant/core/blob/f4cc4a0896358f4c8df5a193d900716fabb7ad32/homeassistant/components/xiaomi_miio/__init__.py#L126

Reply:
Yeh, i guess we had it from [cookiecuter](https://github.com/oncleben31/cookiecutter-homeassistant-custom-component/blob/933ca228d63f5d0c249c9328f16d7f33f6ae1749/%7B%7Bcookiecutter.project_name%7D%7D/custom_components/%7B%7Bcookiecutter.domain_name%7D%7D/__init__.py#L59) where they still use `async_add_job`, but if `async_create_task` is better since it returns `Callable`, as you say, it will probs be good for typing purpose